cluster/examples/kubernetes/ceph/crds.yaml

```diff
@@ -1,415 +1,254 @@
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: PoolSpec represents the spec of ceph pool
+            properties:
+              annotations:
+                additionalProperties:
                   type: string
-                disruptionManagement:
-                  type: object
-                  properties:
-                    machineDisruptionBudgetNamespace:
-                      type: string
-                    managePodBudgets:
-                      type: boolean
-                    osdMaintenanceTimeout:
-                      type: integer
-                    pgHealthCheckTimeout:
-                      type: integer
-                    manageMachineDisruptionBudgets:
-                      type: boolean
-                skipUpgradeChecks:
-                  type: boolean
-                continueUpgradeAfterChecksEvenIfNotHealthy:
-                  type: boolean
-                waitTimeoutForHealthyOSDInMinutes:
-                  type: integer
-                mon:
-                  type: object
-                  properties:
-                    allowMultiplePerNode:
-                      type: boolean
-                    count:
-                      maximum: 9
-                      minimum: 0
-                      type: integer
-                    volumeClaimTemplate:
-                      type: object
-                      x-kubernetes-preserve-unknown-fields: true
-                    stretchCluster:
-                      type: object
-                      nullable: true
+                description: The annotations-related configuration to add/set on each
+                  Pod related object.
+                nullable: true
+                type: object
+                x-kubernetes-preserve-unknown-fields: true
+              compressionMode:
+                description: 'The inline compression mode in Bluestore OSD to set
+                  to (options are: none, passive, aggressive, force)'
+                type: string
+              crushRoot:
+                description: The root of the crush hierarchy utilized by the pool
+                type: string
+              deviceClass:
+                description: 'The device class the OSD should set to (options are:
+                  hdd, ssd, or nvme)'
+                type: string
+              enableRBDStats:
+                description: EnableRBDStats is used to enable gathering of statistics
```

Comment:
This link (included in pool.yaml today) might be beneficial as a description here:
`https://docs.ceph.com/en/latest/mgr/prometheus/#rbd-io-statistics`

Also, `statistics` might more accurately be `prometheus metrics`